test/test_page.rb

```diff
@@ -24,6 +24,10 @@
     assert_equal 'Bilbo-Baggins.md', page.path
     assert_equal :markdown, page.format
     assert_equal @wiki.repo.commits.first.id, page.version.id
+
+    assert_not_nil page.last_version
+    assert_equal page.versions.first.id, page.last_version.id
+    assert page.last_version.stats.files.map{|file_stats| file_stats[0]}.include? page.path
```

Comment:
Thanks. Yes, the confusion was caused by the fact that files are represented as arrays with 4 elements. The following creates a little bit of overhead but might be more legible?
```ruby
files.map{ | file | path, _, _, _, = file; path } 
```
Or simply
```ruby
files.map{ | file | file_path = file.first } 
```